### PR TITLE
DISTPG-389 Release notes 11.15

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release-notes-v11.15.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.yml

### CODE REVIEW

**Review by ImTheKai — APPROVED** [review]
Compared that changes against the 11.14 release and it looks good. Date might to be adjusted in case it slips

**Review by LenzGr — APPROVED** [review]
LGTM - thank you @nastena1606

File: docs/release-notes-v11.15.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+11.15](https://www.postgresql.org/docs/release/11.15/) and includes the
+extended set of extensions supplied with Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.2 | a HA (High Availability) solution for PostgreSQL |
+| [PgAudit](https://www.pgaudit.org/)             | 1.3.3   | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+| [pgAudit set_user](https://github.com/pgaudit/set_user)| 3.0.0 | provides an additional layer of logging and control when unprivileged users must escalate themselves to superusers or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.37    | a backup and restore solution for PostgreSQL       |
+|[pgBadger](https://github.com/darold/pgbadger)   | 11.6     | a fast PostgreSQL Log Analyzer.|
```
Comment (LenzGr) [inline docs/release-notes-v11.15.md]: Please change this version number:
```suggestion
|[pgBadger](https://github.com/darold/pgbadger)   | 11.7     | a fast PostgreSQL Log Analyzer.|
```

**Review by LenzGr — CHANGES_REQUESTED** [review]
Sorry, approved too fast. Please fix the pgbadger version number, then we're good!

**Review by LenzGr — APPROVED** [review]
Thank you, looks good!

**Pull request merged**